base.js

```diff
@@ -99,6 +100,33 @@ module.exports = {
         'prefer-spread': ERROR,
         'require-atomic-updates': OFF, // eslint:recommended
         'require-yield': WARNING, // eslint:recommended
+
+        // Additional Plugins
+        'import/order': [
+            ERROR,
+            {
+                'alphabetize': {
+                    order: 'asc'
+                },
+                'newlines-between': 'always',
+                'groups': [
+                    'builtin',
+                    'external',
+                    'internal',
+                    'parent',
+                    'sibling',
+                    'index'
+                ],
+                'pathGroups': [
+                    {
+                        pattern: 'shared/**',
```

Comment:
looks like something project-specific

Reply:
Agreed, there should not be any specific paths